content/TrueCommand/Clusters.md

```diff
@@ -27,7 +27,7 @@ To see this screen, log in to TrueCommand and click the <span class="iconify" da
 ![ClustersScreen](/images/TrueCommand/2.2/ClustersScreen.png "TrueCommand Clusters Screen")
 
 {{< expand "Option descriptions (click to expand)" "v" >}}
-When no clusters are created, there is a single option on this screen:
+If you have not created a cluster, there is a single option on this screen:
```

Comment:
See comment below line 102 re: table style. Question if table should have only 2 columns, "Setting" and "Description". Value column content is merged with Description.